### ISSUE TITLE
Remove the title from the info icon in signpost component

### ISSUE THREAD

**Issue by yuezhizizhang** [vmware/clarity#3942]
## Describe the bug

For <clr-signpost> component, if you don't specify the custom trigger, an info icon is added. On the info icon, the title is set as below:

<clr-icon shape="info" role="none" title="Info">

An title attribute is specified on the icon. However, there is no where to override it or localize it. 

Please remove this title attribute from the info icon.

## How to reproduce

https://v2.clarity.design/signposts

1.  Mouse over the info icon

## Expected behavior

Either we should be able to customize it or localize it. Otherwise, don't add this title.

## Versions

**App**

* Angular: [8.1.3]
* Node: [10.16.0]
* Clarity: [2.1.1]

**Device:**

* OS: [Windows]
* Browser [chrome, safari]

**Comment by bdryanovski** [comment]
I think better solution here will be to provide a way to localize it than removing it. 

Maybe providing a new commonString key will do the job.

**Comment by bdryanovski** [comment]
With the second look it seems that if you don't implement the trigger we gonna handle this for you.

```html
       <ng-container *ngIf="!useCustomTrigger">
            <button
                type="button"
                class="signpost-action btn btn-small btn-link"
                clrSignpostTrigger>
                <clr-icon shape="info" [attr.title]="commonStrings.keys.info"></clr-icon>
            </button>
        </ng-container>
```

You could change the default text by extending `commonStrings.keys.info`. [Here is how.](https://v2.clarity.design/internationalization)

If you need to have more control or customize every icon title, you will just have to implement the trigger. That seems ok for me.

**Comment by gnomeontherun** [comment]
@bdryanovski is correct, the way to customize is to define your own trigger or localize the `info` text. We do a fallback and it localizes based on common strings. Closing as working as expected.
